configure

```diff
@@ -942,6 +941,13 @@ def configure_node(o):
   else:
     o['variables']['coverage'] = 'false'
 
+  if options.shared:
+    o['variables']['node_target_type'] = 'shared_library'
+  elif options.enable_static:
```

Comment:
elseif?

Reply:
@mihai-iorga `node_target_type` is used in line 1446 below. in here it's just to assign correct value. can you elaborate more about your comment?

Reply:
oh .. sorry :)